src/resources/dashboard.yml

```diff
@@ -56,21 +56,22 @@ dashboard:
     complete:
       type: success
       label: Complete
-      content: You’ve filled out your application. You’re on top _form_!
+      content: You’ve filled out your application. You’re on top form!
 
   team-application:
     na:
       type: na
       label: N/A
       content:
-        You didn’t apply as part of a team. You can [change this](/apply/team) but not for long! Applying as
-        a team means your applications will be reviewed together and either all or none of you will be invited.
-
+        You didn’t apply as part of a team. Applying as a team means your applications will be reviewed together and either all or none of you will be invited.
+      action:
+        title: Submit a team application (while you still can)
```

Comment:
Could we be more specific about when this ends? @moosichu

Reply:
When applications in general close, so end of December.